README.md

```diff
@@ -73,6 +73,94 @@ cache-control: no-cache
 
 You can then use the URL `https://ots.fyi/#1cb08e53-46b9-4f21-bbd9-f1eea1594ad9` to access the secret.
 
+--------
+
+### Detailed usage - OTS
+
+Download the latest stable release depending upon the operating system [here](https://github.com/Luzifer/ots/releases) ([https://github.com/Luzifer/ots/releases](https://github.com/Luzifer/ots/releases))
+
+#### For Windows User -
+
+Download ots binary available at URL  -[https://github.com/Luzifer/ots/releases/download/v0.21.0/ots_windows_amd64.zip](https://github.com/Luzifer/ots/releases/download/v0.21.0/ots_windows_amd64.zip)
+
+![ots2.PNG](images/ots2.png)
+
+Double click to unzip/extract the `ots_windows_amd64.exe` file.
+
+Run/execute the `ots_windows_amd64.exe` file. Notice a command prompt as shown below - 
+
+![ots3.PNG](images/ots3.png)
+
+Open your favourite browser and navigate to `http://localhost:3000`
+
+![ots4.PNG](images/ots4.png)
+
+Type your secret in the input field as shown below `secret123`
+
+![ots5.PNG](images/ots5.png)
+
+Click on **Create the secret!**
+
+![ots6.PNG](images/ots6.png)
+
+- You can share the above URL `http://localhost:3000/#2f6e1abc......` with the person you wanted to share the password with.
```

Comment:
Sharing a `localhost:3000` URL will not get the person you've shared the secret with anywhere. It only exists on your computer.